src/modules/core/components/MemberReputation/MemberReputation.tsx

```diff
@@ -41,7 +41,7 @@ enum ZeroValue {
 
 type PercentageReputationType = ZeroValue | number | null;
 
-const calculatePercentageReputation = (
+export const calculatePercentageReputation = (
   decimalPlaces: number,
```

Comment:
I think the `decimalPlaces` prop should have a default value of `2` since that is about what we use everywhere in the dapp.

And just pass a different value if the display we require is non-standard.

Also, I think this could benefit from being extracted into it's own `util`

Reply:
On the user profile popover, I extracted this to utils: https://github.com/JoinColony/colonyDapp/pull/2685/commits/6fec45db81419366adc33d0be50371dbac26f925

Perhaps Olga can cherry-pick that commit or do something similar?

Reply:
Yeah, I noticed that Armando did it in his PR and I thought that would be fixed during the rebase. But i've cherry-picked the commit, adapted to my code & also made the decimal places optional with a default value.